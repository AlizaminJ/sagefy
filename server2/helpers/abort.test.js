const abort = require('./abort')

describe('#abort', () => {
  test('err with props', () => {
    expect(abort(418, 'hello')).toMatchSnapshot()
  })

  test('err with json', () => {
    expect(abort(418, 'hello', { errors: [] })).toMatchSnapshot()
  })
})
