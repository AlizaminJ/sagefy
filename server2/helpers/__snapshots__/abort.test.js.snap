// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#abort err with json 1`] = `[Error: hello]`;

exports[`#abort err with props 1`] = `[Error: hello]`;
